src/unlink.js

```diff
@@ -89,10 +89,14 @@ module.exports = function unlink(config, args) {
   unlinkDependencyIOS(project.ios, dependency, packageName);
 
   const allDependencies = getDependencyConfig(config, getProjectDependencies());
+  const otherDependencies = filter(allDependencies, ({name}) => name !== packageName);
```

Comment:
I pushed the change to remove destructuring. However, I didn't change the `eslintrc` since it looks like it needs a serious overall given it's using `babel-eslint` which doesn't seem to be affected by me changing `ecmaFeatures`. I'll file an issue about this.

Also, there's a test failure: `makeSettingsPatch should build a patch function`